client/lib/posts/post-list-store.js

```diff
@@ -166,8 +166,9 @@ export default function( id ) {
 		if ( postIds.length ) {
 			// did we actually find any new posts?
 			postIds = difference( postIds, _activeList.postIds );
+
 			if ( postIds.length ) {
-				_activeList.postIds = _activeList.postIds.concat( postIds );
+				_activeList.postIds = postIds.concat( _activeList.postIds );
```

Comment:
Yes, I'll roll back this change and work in some smarter. posts-list has its own way to storing data. it contemplates the cases when the user adds/removes/edits a post.